eng/common/scripts/Generate-DocIndex.ps1

```diff
@@ -0,0 +1,150 @@
+# Generates an index page for cataloging different versions of the Docs
+[CmdletBinding()]
+Param (
+    $DocFx,
+    $RepoRoot,
+    $DocGenDir
+)
+. (Join-Path $PSScriptRoot common.ps1)
+$GetGithubIoDocIndexFn = "Get-${Language}-GithubIoDocIndex"
```

Comment:
This is exactly the change I had in mind when we talked but my expectation is that we wouldn't put the changes under eng/common yet. Instead we should just put the script in each of the language repos and get those changes checked in there and once we feel good about them working there then we can refactor and put this script in eng/common.

Reply:
You mean we put `$GetGithubIoDocIndexFn = "Get-${Language}-GithubIoDocIndex"` in Language-Setting.ps1, right?
Done with the changes. Might look a weird for people when they looked into language-Setting script.

Reply:
No I was meaning to put Generate-DocIndex.ps1 in the language repo not eng/common yet. I was wanting this to be duplicated in every language repo until we have been able to see that it works OK for all those repos and then refactor and move it under eng/common. Once you move this under eng/common then you can split it out into the correct common scripts.

Reply:
Does it overwrite every time we merge sync PR?